lib/action_subscriber/bunny/subscriber.rb

```diff
@@ -15,8 +14,8 @@ def auto_pop!
         # of times we will pop each time we poll the broker
         times_to_pop = [::ActionSubscriber::Threadpool.ready_size, ::ActionSubscriber.config.times_to_pop].min
         times_to_pop.times do
-          queues.each do |queue|
-            delivery_info, properties, encoded_payload = queue.pop(queue_subscription_options)
```

Comment:
It was coming from the `ActionSubscriber::DSL` module which gets mixed into every `ActionSubscriber::Base` class. That is where we currently manage whether a subscriber is going to use message acknowledgements, but now we are managing that in the route so I just moved the method from the module into the `Route` object.